tests/src/test/java/alluxio/client/fs/LoadMetadataIntegrationTest.java

```diff
@@ -174,35 +174,33 @@ public void loadMetadataOnceAfterUfsDelete() throws Exception {
 
   @Test
   public void loadAlwaysConfiguration() throws Exception {
```

Comment:
Whole test goes through FileSystem. So merging should supply the defaults.

Reply:
The issue is that the clients are created after the cluster is started and before we have a chance to set configuration, so these tests need to specifically set the options in order for their values to be respected

Reply:
However after closer inspection I think I changed how this test expected to work.

I've restored the original functionality by adding cluster resource config annotations to those tests

```
  @LocalAlluxioClusterResource.Config(
      confParams = {
          PropertyKey.Name.USER_FILE_METADATA_LOAD_TYPE, "NEVER/ONCE/ALWAYS"
      })
```